dateparser/date.py

```diff
@@ -14,6 +14,12 @@
 from dateparser.languages.detection import AutoDetectLanguage, ExactLanguages
 
 
+APOSTROPHE_LOOK_ALIKE_CHARS = [
+    u'\u2019', u'\u02bc', u'\u02bb', u'\u055a', u'\ua78c',
+    u'\uff07', u'\u2032', u'\u2035', u'\u02b9', u'\uff07',
```

Comment:
Neat! But now I am puzzled... In this case the descriptive name is enough for representation for human beings, but it's still good to have the UTF code nearby (well, actually the `\N{name}` is sufficient in most cases, so even the comment is not too necessary). And the rendered form is easy to find through printing the list... So how about:
```
u'\N{RIGHT SINGLE QUOTATION MARK}',   # u'\u2019'
u'\N{MODIFIER LETTER APOSTROPHE}',    # u'\u02bc'
u'\N{MODIFIER LETTER TURNED COMMA}',  # u'\u02bb'
u'\N{ARMENIAN APOSTROPHE}',           # u'\u055a'
u'\N{LATIN SMALL LETTER SALTILLO}',   # u'\ua78c'
u'\N{PRIME}',                         # u'\u2032'
u'\N{REVERSED PRIME}',                # u'\u2035'
u'\N{MODIFIER LETTER PRIME}',         # u'\u02b9'
u'\N{FULLWIDTH APOSTROPHE}',          # u'\uff07'
```
Doesn't it look pretty? :smile: